frontend/src/components/workflow-list.ts

```diff
@@ -255,7 +255,7 @@ export class WorkflowListItem extends LitElement {
       role="button"
       href=${`/orgs/${this.workflow?.oid}/workflows/crawl/${
         this.workflow?.id
-      }#${this.workflow?.currCrawlId ? "watch" : "artifacts"}`}
+      }#${this.workflow?.lastCrawlState === "running" ? "watch" : "artifacts"}`}
```

Comment:
Since we have additional states besides `running` soon, perhaps we set a `isCrawlRunning` field on the backend, and then have frontend check that?

Reply:
Probably just on basis of these:
https://github.com/webrecorder/browsertrix-cloud/blob/main/backend/btrixcloud/crawls.py#L32
so its set all in one place.

Reply:
I've set `isCrawlRunning` in the backend and am now using it in the frontend. Instead of checking by status, I'm setting it to `True` when the crawl starts (manually or via schedule), and then `False` when we hit `mark_finished` in the operator - which I think should cover all of our bases?

Reply:
I've tested it locally, including with stopped and cancelled crawls, and it is working as expected.